test/e2e/simple/template-config.yaml

```diff
@@ -0,0 +1,23 @@
+tag_specification:
+  namespace: ocp
+  name: "4.5"
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.14
+resources:
+  '*':
+    limits:
+      cpu: 500m
+    requests:
+      cpu: 10m
+tests:
+  - as: success
+    commands: exit 0
+    container:
+      from: os
```

Comment:
Necessary?

Reply:
Yes, otherwise ci-op validation complains - you need to have either `tests` or `images`.

Reply:
Yes, you need a test or images in the config for it to pass validation :)